src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -551,8 +551,9 @@ enum SMART_ACTION
     SMART_ACTION_START_CLOSEST_WAYPOINT             = 113,    // wp1, wp2, wp3, wp4, wp5, wp6, wp7
     SMART_ACTION_RISE_UP                            = 114,    // distance
     SMART_ACTION_RANDOM_SOUND                       = 115,    // soundId1, soundId2, soundId3, soundId4, soundId5, onlySelf
+    SMART_ACTION_SET_CORPSE_DELAY                   = 116,
```

Comment:
Sure, anything you want inserted, I will add it. Looks like the other comments are referencing where it is used or which DB columns are linked / affected by the action.